### ISSUE TITLE
xy-input label attribute / property can't be set dynamically?

### ISSUE THREAD

**Issue by bahrus** [XboxYan/xy-ui#43]
Great library, I like your visual effects.

I may be missing something, but if I open chrome tools to this page:  https://xy-ui.codelabo.cn/docs/#/xy-input, and try to modify the label property / attribute, nothing happens.  Is this by design?  It would be nice to have the ability to change the label.

**Comment by XboxYan** [comment]
yes, some attributes are not dynamically, because it doesn't always need to change 
you can watch it in `attributeChangedCallback`

```js
attributeChangedCallback (name, oldValue, newValue) {
    //
}
```

**Comment by bahrus** [comment]
I am adding your component into a document dynamically, including setting the attribute dynamically.  I think because you are constructing the shadowRoot.innerHTML in the constructor, rather than in connectedCallback, the attribute is not yet established, and isn't getting applied.
